README.md

```diff
@@ -139,10 +139,6 @@ Example command:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219119071-6fbfc1bc-341e-45eb-bf99-a6a9413ec5c3.png)

**[Optional]**

- [ ] It would be nice if you add some frequently asked questions and their answers in the section shown in the image above :arrow_up: to make your readme file more professional :+1: